src/general/OptionsMenu.cs

```diff
@@ -1167,6 +1205,11 @@ private void OnTranslationSitePressed()
         OS.ShellOpen("https://translate.revolutionarygamesstudio.com/engage/thrive/");
     }
 
+    private void UpdateSelectedOutputDevice(Settings settings)
+    {
+        outputDeviceSelection.Selected = OutputDevices.IndexOf(settings.SelectedOutputDevice.Value ?? "Default");
```

Comment:
Doesn't this actually not work because the text "Default" might not be at position 0 where the translation is forcefully applied there on line 750?

Reply:
this works because the `OutputDevices` list is not affected by translations at all. Only the `OptionButton`'s list is translated

Reply:
However, you still apply the default device name translation to item 0, right? Shouldn't that also use the index based lookup, otherwise if the default item is not first (for example in a new Godot version), then the translation is put on the wrong item, right?

Reply:
I've read through the source code of godot and it seems like the order is dependent of the audio driver implementations. My best idea to make sure it is on 0 is to search for the default device and put it on index 0 when calling AufioServer.GetDevices

Reply:
You could do that. 
But I was thinking that you could simply just use the indexOf to find where the default device is and apply the text to that index.

Reply:
I actually prefer the sorting solution. This guarantees the user that the Default device is always at the top of the list

Reply:
Yeah, you can do that, then (if you already did it and I missed the notification about that, sorry, I've gotten so many email from github recently that I haven't read the new commit notifications).